cpp/lib/include/ZoteroHarvesterConfig.h

```diff
@@ -40,6 +42,27 @@ namespace Config {
 static constexpr unsigned DEFAULT_ZEDER_ID(0);
 
 
+
+struct DownloadDelayParams {
+    unsigned default_delay_;
```

Comment:
I'd add a comment to indicate the units or, even better, rename the field to something like default_delay_in_seconds_ or whatever is appropriate.